spec/features/viewing_all_spaces_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'viewing all spaces' do
```

Comment:
In order for this test to pass, you need to add entries using the psql console.  That's a good start but it's a situation which quickly becomes unmanageable.  Imagine what will happen when you have more tests like this and everyone in the team needs to run those tests on their computer.  They'll all have to set things up in exactly the right way.

Instead, you can add some spaces here, in the test, using the `PG` gem.  Then everything we need is right here in the codebase.

Reply:
connection = PG.connect(dbname: 'makers_bnb')
connection.exec('INSERT ...')